mix.exs

```diff
@@ -54,13 +58,16 @@ defmodule Opencensus.Absinthe.MixProject do
       {:absinthe_plug, "~> 1.4.0", optional: true},
       {:credo, "~> 1.1.0", only: :test},
       {:dialyxir, "~> 1.0.0-rc.6", only: :dev, runtime: false},
-      {:ex_doc, ">= 0.0.0", only: :docs},
+      {:ex_doc, ">= 0.0.0", only: [:docs, :test]},
       {:excoveralls, "~> 0.11.1", only: :test},
       {:inch_ex, "~> 2.0.0", only: :docs},
+      {:jason, "~> 1.0", only: [:docs, :test]},
       {:licensir, "~> 0.4.0", only: :test},
       {:mix_test_watch, "~> 0.8", only: :test},
       {:opencensus, "~> 0.9.2"},
-      {:opencensus_elixir, "~> 0.4.0"}
+      {:opencensus_elixir, "~> 0.4.0"},
```

Comment:
In general I prefer to rely on Erlang library itself, as a way to avoid dependency problems, like `opencensus_elixir` lagging behind.

Reply:
I'm with you, but I'm also averse to copy-paste maintenance. The only alternative is dragging the responsibility to the core package, which has had response time problems in the past and excludes any help from Elixir developers. Strikes me as especially odd to do when the problem is _had_ by the Elixir developers e.g. `Logger.metadata` sync and friendlier ways to get to the contents of the records. Some of that'll go away depending on @tsloughter's new behaviour design. Some won't.